docs/reference/supported-environments.md

```diff
@@ -32,14 +32,14 @@ We highly recommend running Camunda Platform 8 Self-Managed in a Kubernetes envi
 
 Requirements for the components can be seen below:
 
-| Component                | Java version | Other requirements                                                                       |
-| ------------------------ | ------------ | ---------------------------------------------------------------------------------------- |
-| Zeebe Broker and Gateway | OpenJDK 17+  | Elasticsearch 7.16.x (only if Elastic exporter is used)                                  |
-| Operate                  | OpenJDK 11+  | Elasticsearch 7.16.x                                                                     |
-| Tasklist                 | OpenJDK 11+  | Elasticsearch 7.16.x                                                                     |
-| Identity                 | OpenJDK 17+  | Keycloak 16.1.1, 19.0.3                                                                  |
-| Optimize                 | OpenJDK 11+  | Elasticsearch 7.13.x - 7.15.x, 7.16.2+, 7.17.x                                           |
-| Web Modeler (Beta)       | -            | Keycloak 16.1.1<br/>PostgreSQL 14.x (other database systems are currently not supported) |
+| Component                | Java version | Other requirements                                                                               |
+| ------------------------ | ------------ | ------------------------------------------------------------------------------------------------ |
+| Zeebe Broker and Gateway | OpenJDK 17+  | Elasticsearch 7.16.x (only if Elastic exporter is used)                                          |
+| Operate                  | OpenJDK 11+  | Elasticsearch 7.16.x                                                                             |
+| Tasklist                 | OpenJDK 11+  | Elasticsearch 7.16.x                                                                             |
+| Identity                 | OpenJDK 17+  | Keycloak 16.1.1, 19.0.3                                                                          |
+| Optimize                 | OpenJDK 11+  | Elasticsearch 7.13.x - 7.15.x, 7.16.2+, 7.17.x                                                   |
+| Web Modeler (Beta)       | -            | Keycloak 16.1.1, 19.0.3<br/>PostgreSQL 14.x (other database systems are currently not supported) |
```

Comment:
:x: Earlier we had an issue with Keycload 16.1.1 https://github.com/camunda-cloud/identity/issues/1413

@Ben-Sheppard Did you fix the issue for 16.1.1? I was unsure after looking at your fix whether it also solves the issue for 16.1.1

Reply:
I just copied the values from Identity that also states compatibility with both Keycloak versions. And isn't it rather an Identity issue anyway (if you can't copy a secret from their UI)? The Web Modeler itself is definitely compatible with Keycloak 16.1.1 and 19.0.3 :).

Reply:
:smile:  If someone wants to test Web Modeler self managed, and it doesn't work (for whatever reason), we will be called :upside_down_face: 

But ok, seems like Ben is already on holidays. So feel free to merge it :santa:

Reply:
By the way, the latest stable version of Identity (`8.1.5`) – which is also used in the Docker Compose file for Self-Managed – does not support M2M applications yet anyway, so users won't get the chance to run into the issue 🙃.

Reply:
Ah, I forgot the release cycle :see_no_evil: